js/classroom.js

```diff
@@ -0,0 +1,207 @@
+/**
+ * A class representing classrooms.
+ * 
+ * @Constructor
+ *
+ * @example
+ *  var classroomExample = {
+ *    classroomCode: "43",
+ *    selected: 1,
+ *    horas_aula: 0,
+ *    vagas_ofertadas: 0,
+ *    vagas_ocupadas: 0,
+ *    alunos_especiais: 0,
+ *    saldo_vagas: 0,
+ *    pedidos_sem_vaga: 0,
+ *    teachers: [
+ *      "First Guy",
+ *      "Second Son",
+ *      "third Weird"
+ *    ],
+ *    schedules: [{@link Schedule}],
+ *    htmlElement: div.classroom-info
+ *  }
+ *
+ * @see UI#createClassroomInfo
+ */
+ // IMPORTANT: the 'ui' variable must be already set up!
+function Classroom(jsonObj, parentLecture) {
+  this.parent = parentLecture;
+  this.teachers = new Array();
+  this.schedules = new Array();
+  if (jsonObj) {
+    this.classroomCode = jsonObj.classroomCode;
+    this.horas_aula = jsonObj.horas_aula;
```

Comment:
Multilanguage code. :P Is it possible to map all properties to be either all in English or all in Portuguese?